lib/cli-engine.js

```diff
@@ -193,24 +198,40 @@ function processText(text, configHelper, filename) {
     }
 
     if (processor) {
+        debug("Using processor");
         var parsedBlocks = processor.preprocess(text, filename);
         var unprocessedMessages = [];
         parsedBlocks.forEach(function(block) {
             unprocessedMessages.push(eslint.verify(block, config, filename));
         });
         messages = processor.postprocess(unprocessedMessages, filename);
+
+        // TODO(nzakas): Figure out how fixes might work for processors
```

Comment:
I'm actually not sure how to make this work. I can fix the individual `block`s as they come through, but I have zero idea how to update the overall text. The problem is that I don't have enough information to know how to update the complete text since it's the processor that parses it and passed back just the blocks to lint. I have no idea how to re-insert those blocks into the text at the end.

The `postprocess()` just purely deals with messages.

At this point, it might be better to just omit fixing from processed text until later. We might need to rethink how we do processors in order to make it work.